rust/tools/authorization-logic/src/compilation_top_level.rs

```diff
@@ -38,9 +38,11 @@ fn source_file_to_ast(filename: &str, in_dir: &str) -> AstProgram {
     astconstructionvisitor::parse_program(&source[..])
 }
 
-pub fn compile(filename: &str, in_dir: &str, out_dir: &str) {
+pub fn compile(filename: &str, in_dir: &str, out_dir: &str,
+            decl_skip: &Option<Vec<String>>) {
```

Comment:
It is straightforward to pass a string to a Rust library call from `C++`. However, I am not sure how we can do the same for rust vectors. 
Can we have a version where `decl_skip` is a comma-separated or newline-separated list of declarations?  e.g., 
`blah, some, foo`
or
```
blah
some
foo
```

Reply:
That's how the main function already works. It's just the compile function that takes a vector. I guess I can move the bit in the main that converts the string into compile.

Reply:
OK fixed. PTAL @bgogul